### PR TITLE
[Logstash] Add pipeline info to pipeline screen

### PR DESCRIPTION
**Pull request by robbavey**
## Proposed commit message

This commit adds pipeline information to the Pipelines View. This involves retrieving the `info` object from the `pipelines` view to retrieve `ephemeral_id`, `workers`, `batch_size` and `batch_delay` for each pipeline. 

Additionally, this commit adds a table to the Pipelines View to show `workers`, `batch_size`, `batch_delay`, the type of queue being used, and the current size of the queue if it is a persistent queue

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

Install the integration, and point to a recent version of Logstash. Verify that the Pipelines View shows the correct data for each pipeline

## Screenshots

Screenshot 2024-06-13 at 12 55 23 PM

### CODE REVIEW

File: packages/logstash/kibana/dashboard/logstash-a42d7060-45e6-11ee-957b-3720c0b0fbc5.json [hunk 1]
```diff
@@ -1761,7 +1761,7 @@
     "coreMigrationVersion": "8.8.0",
     "created_at": "2024-06-05T19:53:56.470Z",
     "id": "logstash-a42d7060-45e6-11ee-957b-3720c0b0fbc5",
-    "managed": false,
+    "managed": true,
```
Comment (jennypavlova) [inline packages/logstash/kibana/dashboard/logstash-a42d7060-45e6-11ee-957b-3720c0b0fbc5.json:1764]: Why is this change needed?

File: packages/logstash/data_stream/pipeline/agent/stream/cel.yml.hbs [hunk 2]
```diff
@@ -21,9 +21,17 @@ redact:
 
 
 program: |
-  get(state.url).as(resp, bytes(resp.Body).decode_json().as(body,
+  get(state.url + "/stats?graph=true&vertices=true").as(resp, bytes(resp.Body).decode_json().as(body,
     body.pipelines.map(pipeline_name, pipeline_name != ".monitoring-logstash", {
       "name": pipeline_name,
+      "info": get(state.url + "/pipelines/" + pipeline_name).as(resp,
+        bytes(resp.Body).decode_json().as(pipes,
+          has(pipes.pipeline) && has (pipes.pipelines) ?
```
Comment (jennypavlova) [inline packages/logstash/data_stream/pipeline/agent/stream/cel.yml.hbs:29]: Do we need both checks here?

**Review by jennypavlova — COMMENTED** [review]
Great job 💯 Left some questions and I am trying to test the change. I am using elastic-package to build the integration and `elastic-package stack up -d` but I am not sure that this will work with Logstash - I installed the integration in Kibana (latest version 2.4.10) and enabled self-monitoring. Do you know if I need to do something else to see the changes?

File: packages/logstash/kibana/dashboard/logstash-a42d7060-45e6-11ee-957b-3720c0b0fbc5.json (hunk 1 above)
Comment (robbavey) [reply packages/logstash/kibana/dashboard/logstash-a42d7060-45e6-11ee-957b-3720c0b0fbc5.json:1764]: My understanding is that all of the dashboards we provide should be managed, and this was an oversight

File: packages/logstash/data_stream/pipeline/agent/stream/cel.yml.hbs (hunk 2 above)
Comment (robbavey) [reply packages/logstash/data_stream/pipeline/agent/stream/cel.yml.hbs]: Removed the first one

**Review by jennypavlova — APPROVED** [review]
Thanks for adding the changes! 
Probably my local setup is not working correctly because I don't see any values in the dashboards. The code looks good to me. 🎉 

> This should be able to be tested using elastic-package stack up -d, with stack.logstash_enabled: true setup in the config.yml to bring up a logstash. Then setting up the integration with the agent pointing to localhost:9600, and the "technical preview" metrics radio button checked should install the dashboards, and have new entries showing in "Pipelines Overview"

I built the package and I followed the steps but maybe I missed something:


https://github.com/elastic/integrations/assets/14139027/847a29ce-2f07-4886-8a84-0bac2d59e6ef



And http://localhost:9600/_node returns: 
image

**Comment by robbavey** [comment]
Apologies - if you are using the `elastic-package` agent, the integration should point to `http://logstash:9600`, rather than `http://localhost:9600` - if you change it, the values should appear after a short while

**Comment by jennypavlova** [comment]
@robbavey Thank you for the instructions! I did that and with `http://logstash:9600` I see the data 🎉
